BasedOnDiagram.sql

```diff
@@ -0,0 +1,60 @@
+CREATE TABLE "patients" (
+  "id" int,
+  "name" varchar(30),
+  "date_of_birth" date,
+  PRIMARY KEY ("id")
+);
+
+CREATE TABLE "medical_histories" (
+  "id" int,
+  "admitted_at" timestamp,
+  "patient_id" int,
+  "status" varchar(40),
+  PRIMARY KEY ("id"),
+  CONSTRAINT "FK_medical_histories.patient_id"
+    FOREIGN KEY ("patient_id")
+      REFERENCES "patients"("id")
+);
+
+CREATE TABLE "invoices" (
+  "id" int,
+  "total_amount" decimal,
+  "generated_at" timestamp,
+  "payed_at" timestamp,
+  "medical_history_id" int,
```

Comment:
- [ ] This is a foreign key so kindly consider setting it up as a foreign key. it references medical_histories(id). consider fixing it.